### DISCUSSION TITLE
tj/tj-opensource-affine

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#13] [Announcements]
# tj/tj-opensource-affine

Notion这款笔记软件相信很多开发者都比较熟悉了，很多读者，包括我自己都用它来记录和管理自己的笔记。今天给大家推荐一个最近比较火的开源替代方案：AFFiNE。目前该开源项目已经斩获20.1K Star，热度非常的高，下面一起来认识一下这个继Notion之后，被热捧的开源软件吧。 通过官方首页的介绍，我们可以知道，AFFiNE是一款集写作、绘画、计划管理与一体的软件。对标的是Notion、Miro、Monday这些热门当下热门的笔记软件。

https://www.didispace.com/tj/tj-opensource-affine.html

**Comment by huiyonghkw** [comment]
UI是第一选择

**Comment by Wolfdebug** [comment]
和思源笔记有啥区别嘛
